indico/modules/events/payment/models/transactions.py

```diff
@@ -253,6 +253,6 @@ def create_next(cls, registration, amount, currency, action, provider=None, data
         except DoublePaymentTransaction:
             next_status = TransactionStatus.successful
             double_payment = True
-            Logger.get('payment').warning("Received successful payment for an already paid registration")
+            Logger.get('payment').info("Received successful payment for an already paid registration")
         new_transaction.status = next_status
         return new_transaction, double_payment
```

Comment:
We could just return the new transaction now

Reply:
I hesitated about removing it, since it may be useful in other places. But yeah, maybe it's best to keep it minimal.